docs/common/craft-parts/explanation/overlay_parameters.rst

```diff
@@ -4,13 +4,13 @@ Overlay Parameters
 ------------------
 
 A part has three parameters that can be used to adjust how the overlay step
-works: ``overlay-packages``, ``overlay-script`` and ``overlay-filter``.
```

Comment:
I don't see any current usages of `overlay-filter` in our org, but if you believe that was ever the case, I'd probably suggest adding some sort of note about this change

Reply:
Inspecting the git history, it does seem this was a an old typo, as the field name is "overlay-files" and its alias is "overlay". I am going to put it as "overlay" in the docs, as aliases are more preferrable to use.